Justfile

```diff
@@ -48,7 +50,15 @@ build-generate:
 
 # Build command-line tools
 build +tools: build-protos build-zips build-frontend
+  @just build-without-frontend $@
```

Comment:
Do we need to include the `pnpm-install` dependency here?

Reply:
I don't think so, because build-protos is often a no-op, and if it actually does something it calls 'build-protos-unconditionally', which already has the pnpn-install dependency.

This optimization saves a bit of time, as there is no pnpn involved in most of the tests.